content/id/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -0,0 +1,72 @@
+---
+reviewers:
+- davidopp
+- lavalamp
+title: Ikhtisar Administrasi Kluster
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+Ikhtisar administrasi kluster ini ditujukan untuk siapapun yang akan membuat atau mengelola kluster Kubernetes.
+Diharapkan untuk memahami beberapa [konsep](/docs/concepts/) dasar Kubernetes sebelumnya.
+{{% /capture %}}
+
+{{% capture body %}}
+## Perencanaan Kluster
+
+Lihat panduan di [Memilih Solusi yang Tepat](/docs/setup/pick-right-solution/) untuk mempelajari beberapa contoh tentang bagaimana merencanakan, mengatur dan mengonfigurasi kluster Kubernetes. Solusi yang akan dipaparkan di bawah ini disebut *distro*.
+
+Sebelum memilih panduan, berikut adalah beberapa hal yang perlu dipertimbangkan:
+
+ - Apakah kamu hanya ingin mencoba Kubernetes pada komputermu, atau kamu ingin membuat sebuah kluster dengan *high-availability*, *multi-node*? Pilihlah distro yang paling sesuai dengan kebutuhanmu.
+ - **Jika kamu merencanakan kluster dengan *high-availability***, pelajari bagaimana cara mengonfigurasi [kluster pada *multiple zone*](/docs/concepts/cluster-administration/federation/).
+ - Apakah kamu akan menggunakan **Kubernetes kluster di *hosting***, seperti [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/), atau ***hosting* sendiri klustermu**?
```

Comment:
*hosting* format doesnt work out

similar to the rest of the stuffs, review here: https://deploy-preview-14156--kubernetes-io-master-staging.netlify.com/id/docs/concepts/cluster-administration/cluster-administration-overview/#perencanaan-kluster